pkg/resource/k8s-events-config/alloy/events-logger.alloy.template

```diff
@@ -0,0 +1,27 @@
+loki.source.kubernetes_events "local" {
+	namespaces = {{ .ScrapedNamespaces }}
```

Comment:
What is that .ScrapedNamespaces ?

Reply:
They're the namespaces for which alloy (or grafana-agent) will scrape the events, so here the default namespaces. But you have a point here : I forgot to write it in in the `generateAlloyConfig` function :sweat_smile:

Reply:
So I added the `ScrapedNamespaces` field and used the same code as for the grafana-agent to define this so I chose to move this piece of code into a function in common.